.github/workflows/ci-sql-cli.yaml

```
@@ -90,7 +90,7 @@ jobs:
             ~/.cache/pip
             sql-cli/.nox
             .local/
-          key: ${{ runner.os }}-2.3-${{ hashFiles('sql-cli/pyproject.toml') }}
+          key: ${{ runner.os }}-${{ hashFiles('sql-cli/pyproject.toml') }}
```

```suggestion
          key: sql-cli-tests-${{ runner.os }}-${{ hashFiles('sql-cli/pyproject.toml') }}
```

can we add the job name prefix like suggested to the key.